site/content/en/docs/user/kwok-in-local.md

```diff
@@ -6,8 +6,33 @@ This document walks you through how to run `kwok` in your local.
 
 {{< /hint >}}
 
-After you have [`kwok` installed]({{< relref "/docs/user/install" >}}) in a local computer,
-you can run `kwok` immediately for clusters and maintain their heartbeats:
+## Create clusters
```

Comment:
``` readme

# Running `kwok` out of cluster

{{< hint "info" >}}

This document walks you through how to run `kwok` out of cluster for a Kubernetes cluster.

{{< /hint >}}

## Prerequisites

Firstly, we need to have a Kubernetes cluster, and the `kwok` command-line tool must be available in your `PATH`.

## Running

Next, we can run `kwok` out of cluster by using the `--kubeconfig` flag.

```bash
kwok \
  --kubeconfig=~/.kube/config \
  --manage-all-nodes=false \
  --manage-nodes-with-annotation-selector=kwok.x-k8s.io/node=fake \
  --manage-nodes-with-label-selector= \
  --disregard-status-with-annotation-selector=kwok.x-k8s.io/status=custom \
  --disregard-status-with-label-selector= \
  --cidr=10.0.0.1/24 \
  --node-ip=10.0.0.1
\```

Finally, we can see the `kwok` is running out of cluster for the Kubernetes cluster.

## Next Steps

Now, we can use `kwok` to [manage nodes and pods] in the Kubernetes cluster.

[manage nodes and pods]: {{< relref "/docs/user/kwok-manage-nodes-and-pods" >}}
```

Reply:
Change the path to `kwok-out-cluster` ?